counter_robots/robot_detection.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of COUNTER-Robots.
+# Copyright (C) 2018 CERN.
+#
+# COUNTER-Robots is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Robot and machine detection."""
+
+import re
+
+from counter_robots import _get_resource_content
```

Comment:
Damn, last fix: since the whole package is basically 6 functions and some configuration, they can all live in `__init__.py` (in order also to avoid importing "underscore" functions). So basically you can add the contents of all the *.py files in `__init__.py`